CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 TiUP Changelog
 
+## [1.3.0] 2020.12.17
+
+### New Features
+
+- Modify TiFlash's query memory limit from 10GB to 0(unlimited) in tiup-playground ([#907](https://github.com/pingcap/tiup/pull/907), [@LittleFall](https://github.com/LittleFall))
+- Import configuration into topology meta when migrating a cluster from Ansible ([#766](https://github.com/pingcap/tiup/pull/766), [@yuzhibotao](https://github.com/yuzhibotao))
+  - Before, we stored imported ansible config in ansible-imported-configs which is hidden for users, in this release, we merge the configs into meta.yaml so that the user can see the config with the command `tiup cluster edit`
+- Enhance the `tiup mirror` command ([#860](https://github.com/pingcap/tiup/pull/860), [@lucklove](https://github.com/lucklove))
+  - **Support merge two or more mirrors into one**
+  - Support publish component to local mirror besides remote mirror
+  - Support add component owner to local mirror
+- Support deploy cluster with hostname besides ip address (EXPERIMENTAL) ([#948](https://github.com/pingcap/tiup/pull/948),[#949](https://github.com/pingcap/tiup/pull/949), [@fln](https://github.com/fln))
+- Support setting custom timeout for waiting instances up in tiup-playground ([#968](https://github.com/pingcap/tiup/pull/968), [@unbyte](https://github.com/unbyte))
+- Support check and disable THP in tiup cluster check ([#964](https://github.com/pingcap/tiup/pull/964), [@anywhy](https://github.com/anywhy))
+- Support sign remote manifest and rotate root.json ([#967](https://github.com/pingcap/tiup/pull/967), [@lucklove](https://github.com/lucklove))
+
+### Fixes
+
+- Fix the issue that public key created by TiUP not removed after the cluster destroyed ([#910](https://github.com/pingcap/tiup/pull/910), [@9547](https://github.com/9547))
```

Comment:
Fixed the issue that the public key created by TiUP was not removed after the cluster was destroyed